app/models/paper.rb

```diff
@@ -55,4 +55,20 @@ def valid_methodologies
       'case report'
     ]
   end
+
+  # Skips validations, fallthrough to autosave methods for easy import
+  def validate_associated_records_for_authors; true; end
+  def validate_associated_records_for_links; true; end
```

Comment:
Add links when they create a paper, correct? If it's nested in the paper, that would cause a problem. I was experimenting with a way to tell the validations if it was an import or not, but it got too involved for this PR.